hack/e2e.sh

```diff
@@ -23,7 +24,16 @@ runTest()
 	fi
 }
 
-go get -u github.com/thecodeteam/gocsi/mock
+#
+# TODO Once 0.2.0 change gets merged into gocsi repo, switch back to "go get"
+#
+git clone https://github.com/thecodeteam/gocsi $GOPATH/src/github.com/thecodeteam/gocsi
+pushd $GOPATH/src/github.com/thecodeteam/gocsi
+git checkout $CSI_MOCK_VERSION
+make build
```

Comment:
Does this copy it to `$GOPATH/bin`?

Reply:
Yes, otherwise e2e test would have filed.